distrobuilder/vm.go

```diff
@@ -103,6 +103,34 @@ func (v *vm) mountImage() error {
 
 	v.loopDevice = strings.TrimSpace(stdout)
 
+	// Ensure the partitions are accessible. This part is usually only needed
+	// if building inside of a container.
+
+	out, err := lxd.RunCommand("lsblk", "--raw", "--output", "MAJ:MIN", "--noheadings", v.loopDevice)
+	if err != nil {
+		return err
+	}
+
+	deviceNumbers := strings.Split(out, "\n")
+
+	if !lxd.PathExists(v.getUEFIDevFile()) {
+		fields := strings.Split(deviceNumbers[1], ":")
+
+		err := shared.RunCommand("mknod", v.getUEFIDevFile(), "b", fields[0], fields[1])
+		if err != nil {
+			return err
+		}
+	}
+
+	if !lxd.PathExists(v.getRootfsDevFile()) {
+		fields := strings.Split(deviceNumbers[2], ":")
+
+		err := shared.RunCommand("mknod", v.getRootfsDevFile(), "b", fields[0], fields[1])
+		if err != nil {
+			return err
+		}
```

Comment:
How about using unix.Mknod instead of shelling out?

Reply:
I had that at first, but then decided to do it the lazy way. I'll fix that ;)